CRM/Civisocial/BAO/CivisocialUser.php

```diff
@@ -74,14 +74,14 @@ public static function create($params) {
    * @param array $userInfo
    *
    * @return int
-   *        Contact ID of created or existing contact
+   *   Contact ID of created or existing contact
    */
   public static function createContact($userInfo) {
     $email = CRM_Utils_Array::value("email", $userInfo);
     $contacts = civicrm_api3(
-    'contact',
-    'get',
-    array("email" => $email)
+      'contact',
+      'get',
+      array("email" => $email)
     );
 
```

Comment:
i am more comfortable in using dedupe rule to find contact from table. Since there might be contacts with same email but for different contacts and some might be soft deleted.